src/lib/layout/Sidebar.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+  import NavItem from '$lib/navbar/NavItem.svelte';
+  import Icon from 'svelte-icons-pack/Icon.svelte';
+  import FaBars from 'svelte-icons-pack/fa/FaSolidBars';
+  import FaTimes from 'svelte-icons-pack/fa/FaSolidTimes';
+
+  let selectedIndex: number | null = null;
+
+  const items = ['Início', 'Eventos', 'Projetos', 'Equipa', 'Contactos'];
+
+  let sidebarOpen = false;
+</script>
+
+{#if !sidebarOpen}
```

Comment:
```suggestion
{#if sidebarOpen}
```
Avoid negative ifs when there is no difference